### ISSUE TITLE
[E4] E4-1 test 48

### ISSUE THREAD

**Issue by cotefon** [franciscogazitua/IIC2113-2024#120]
Hola! tengo una duda con el test 48, ya que ambos personajes tienen el hechizo Hliðskjálf y se le aplica a ambos el efecto cuando una de las condiciones es partir el combate.

Hliðskjálf: Si la unidad inicia el combate con magia contra un rival que usa magia, el rival no podrá
contraatacar.

En el testcase,  debería decir: Célica no puede hacer un follow up?

![image](https://github.com/franciscogazitua/IIC2113-2024/assets/37127700/b0e112d6-8e0b-4033-abb3-f90c057f808d)

**Comment by RodrigoToroIcarte** [comment]
No. Dado que Celica no puede contraatacar no se anuncia que no puede hacer follow up (porque sabemos que no puede desde el inicio de la batalla). La información nueva es que Robin no pudo hacer un follow up. Por eso se anuncia que _Robin no puede hacer un follow up_.

... de todas formas, admito que esta regla es super arbitraria de nuestra parte.

**Comment by cotefon** [comment]
¿Por qué Robin recibe el efecto cuando Célica no parte el combate?

**Comment by RodrigoToroIcarte** [comment]
Creo que no entendí la pregunta. La habilidad dice "_Si la unidad inicia el combate con magia contra un rival que usa magia, el rival no podrá contraatacar._" Como Robin parte el combate, Celica no puede contraatacar.

**Comment by cotefon** [comment]
Lo que me genera duda es el tema de que imprima que Robin no puede realizar follow up, cuando le correspondía a Célica realizarlo (tiene más spd y la dif entre ambas es 6)

**Comment by RodrigoToroIcarte** [comment]
Celica no puede hacer follow up porque no puede contraatacar. El no poder contraatacar también incluye que no pueda hacer follow up.

El mensaje de que Robin no puede hacer follow up es algo distinto (aunque relacionado). 

![Screenshot from 2024-06-21 17-05-19](https://github.com/franciscogazitua/IIC2113-2024/assets/7000518/9398e004-d785-4e46-83c3-b18432fd99ae)

**Comment by cotefon** [comment]
aaah no había visto eso! muchas gracias!
